### PR TITLE
Controller release

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: controller/src/Main.hs [hunk 1]
```diff
@@ -28,16 +29,16 @@ import qualified Servant.Server as Servant
...
-readEnvDefault d = liftM (fromMaybe d) . readEnv
+readEnvDefault :: Read a => a -> String -> IO a
+readEnvDefault d = fmap (fromMaybe d) . readEnv
 
 main :: IO ()
 main = do
-    port <- readEnvDefault 8080 "PORT"
-    putStrLn $ "Starting on http://localhost:" ++ (show port)
+    port <- readEnvDefault 8080 "RT_PORT"
+    putStrLn $ "Starting on http://localhost:" <> show port
```
Comment (zimbatm) [inline controller/src/Main.hs]: why is it better to use `<>` rather than `++` ?

File: controller/src/Main.hs [hunk 2]
```diff
@@ -28,16 +29,16 @@ import qualified Servant.Server as Servant
 import qualified Servant.Utils.StaticFiles as Servant
 import qualified Text.Blaze.Renderer.Utf8 as Blaze
 
-readEnv :: (Read a) => String -> IO (Maybe a)
-readEnv = liftM (join . fmap readMaybe) . lookupEnv
+readEnv :: Read a => String -> IO (Maybe a)
+readEnv = fmap (join . fmap readMaybe) . lookupEnv
```
Comment (zimbatm) [inline controller/src/Main.hs]: `fmap` is more generic?

Comment (nmattia) [reply controller/src/Main.hs]: yeah, it's not limited to monads, and is more idiomatic

File: controller/src/Main.hs (hunk 1 above)
Comment (nmattia) [reply controller/src/Main.hs]: `++` only works on lists, which are monoids (semigroups) so might as well use a single function for everything (`mconcat`)

**Pull request merged**